scripts/zones/Lufaise_Meadows/npcs/qm_kurrea.lua

```diff
@@ -14,12 +14,13 @@ entity.onTrade = function(player, npc, trade)
         npcUtil.popFromQM(player, npc, ID.mob.KURREA)
     then
         -- Adamantoise Soup
+        player:messageSpecial(ID.text.KURREA_SPAWN_TEXT, 5210)
```

Comment:
add the item to items.lua global, use KURREA_TEXT for 1 text ID and the other messages can be +/- from it. They will always be grouped in the client so shifts won't split them up.